how-to-create-a-theme.md

```diff
@@ -6,40 +6,40 @@ If you make a theme for the first time, refer to the existing theme. Copy the **
 
 ## ⛔ Caution ⛔
 
-Place the theme you created in the Theme folder inside the UserData directory, for roaming this is located at **%APPDATA%\\FlowLauncher\\Themes\\** (AppData Roaming path) and for portable it is by default **%localappdata%\\FlowLauncher\app-\\<VersionOfYourFlowLauncher\>\\UserData\\Themes\\** (AppData Local path). Flow will read from the UserData directory for custom themes and its own app directory for default themes, so make sure you do not place in the location outside of UserData because it will be erased along with the default theme files after an update.
+Place the theme you created in the Theme folder inside the UserData directory, for roaming this is located at `%APPDATA%\FlowLauncher\Themes\` (AppData Roaming path) and for portable it is by default `%localappdata%\FlowLauncher\app-<VersionOfYourFlowLauncher>\UserData\Themes\` (AppData Local path). Flow will read from the UserData directory for custom themes and its own app directory for default themes. Make sure you do not place in the location outside UserData because it will be erased along with the default theme files after an update.
 
 ## Theme elements
 
-The theme file allows you to set the following parts. Each style has a key, and there are items that can be modified. If theme have a key that is not described in this document, we recommend you not to modify it separately.
+The theme file allows you to set the following parts. Each style has a key, and there are items that can be modified. If theme has a key not described in this document, we recommend you not to modify it separately.
 
 (*There is a possibility of changing/deleting this part depending on the version.*)
 
-<img src="https://github.com/Flow-Launcher/docs/raw/main/assets/themelayout.png" style="zoom:50%;" />
+![Flow Launcher screenshot](https://cdn.jsdelivr.net/gh/Flow-Launcher/docs@main/assets/themelayout.png)
 
-</br>
+<br>
 
 ### WindowBorderStyle
 
 In this item, you can set the color, border size, border color, and corner radius of the basic window.
 
-```
+```xml
 <Style x:Key="WindowBorderStyle" BasedOn="{StaticResource BaseWindowBorderStyle}" TargetType="{x:Type Border}">
         <Setter Property="BorderThickness" Value="2" />
         <Setter Property="BorderBrush" Value="#6c7279" /> 
         <Setter Property="CornerRadius" Value="5" />
         <Setter Property="Background" Value="#303840" />
  </Style>
- ```
+```
 
-Window border thickness is recommended from 1 or 2. Corner radius recommends 0, 5 or less.
+Window border thickness is recommended from 1 or 2. Recommended corner radius is 0, 5 or less.
 
-</br>
+<br>
```

Comment:
Incorrect syntax in tag usage

Reply:
does this compile? isn't we need a closed tag? or like `<br/>`? 🤔

Reply:
This would be required in XHTML or XML, not in HTML. HTML defines a list of void elements, like `<input>`, `<link>`, `<img>`, or `<br>`. The browser knows they can't contain anything inside, so it closes them immediately. You can put a slash at the end of them, but it doesn't do anything. More info: https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags

Reply:
interesting